src/app/core-ui/main/announce-notification/announcement-notification.model.ts

```diff
@@ -1,5 +1,5 @@
 
-export class ReleaseNotification {
+export class AnnounceMentNotification {
```

Comment:
s/AnnounceMentNotificationComponent/AnnouncementNotificationComponent/g

There's a weird capital letter 'M' in 'Announcement'